src/code-block.js

```diff
@@ -8,22 +8,32 @@ const preload = (content, resourcePath) => {
 
   const $ = cheerio.load(content, shared.cheerioConfig);
   const codeBlocks = $('stache-code-block');
+  const codeTags = $('stache-code');
 
-  if (!codeBlocks.length) {
+  if (!codeBlocks.length && !codeTags.length) {
     return content;
   }
 
   codeBlocks.each((idx, elem) => {
     const $elem = $(elem);
-    let content = $elem.html()
-      .toString()
-      .replace(/{/g, `{{ '{' }}`)
-      .replace(/</g, '&lt;');
+    replaceCharacters($elem);
+  });
 
-    $elem.html(content);
+  codeTags.each((idx, elem) => {
+    const $elem = $(elem);
+    replaceCharacters($elem);
   });
 
   return $.html().toString();
 };
 
+const replaceCharacters = ($elem) => {
```

Comment:
I'd prefer to put this method in the **utils** directory and import it into separate **code-block.js** and **code.js** files, if possible.

Reply:
Yeah I thought about that a bit. My reasons behind keeping it this way was 
A) the `replaceCharacters` method is looking for specific characters to replace with other specific characters.  It's unlikely to ever be used outside of this use case, so it's not really a shared util.

B) Code vs Code block,  I didn't feel like the file `code.js` would be easily understood at a glance.  the `<stache-code>` element is also still a code block of sorts so I thought it might still fit in with this file.

C) I can't think of any other case in which we would really want to replace those characters as is and have them printed to the page as raw text like we do in these two elements, and the code seemed small enough not to warrant needing another plugin just for those few lines.

I could see a compromise here of coming up with a name that would incorporate both the `stache-code` and `stache-code-block` elements and leave it as one file.  Thoughts?